database/main.py

```diff
@@ -0,0 +1,31 @@
+# main.py
+
+import sqlalchemy as sa
+
+from fastapi import FastAPI
+from sqlalchemy.orm import sessionmaker
+
+from config import DATABASE_URL
+from release_dao import readRelease, getkeys
+
+engine = sa.create_engine(DATABASE_URL)
+Session = sessionmaker(bind=engine)
+
+s = Session()
+
+app = FastAPI()
+
+
+@app.get("/releases")
+async def getall():
+    objectString = "{ "
+    keylist = getkeys()
+    for id in keylist:
+        objectString += readRelease(id)
+    objectString += " }"
+
+    return {"message": objectString}
+
+
+s.commit()
+s.close()

```

Comment:
Again, let us remove all sqlalchemy/session related references from our Web Server layer.
This model should abstract what happens in other layers of our system to keep the SOC (Separation of Concerns).